cookbook/bundles/dependencies.rst

```diff
@@ -0,0 +1,97 @@
+.. index::
+   single: Bundle; Dependencies
+
+How to Use Bundle Dependencies to Load other Bundles
+====================================================
+
+.. versionadded:: 2.8
+    Support for bundle dependencies was introduced in Symfony 2.8.
+
+When working on your own bundle(s), you'll sometimes have the need to reuse
+other bundles, either by just requiring, overriding or inheriting from them.
+
+While Composer takes care about making sure these dependencies are loaded,
+you'll also need to enable these bundles in the kernel.
+
+If your bundle is meant to be reused, bundle dependencies will complicate
+your installation documentation. This makes installation and upgrading your
+bundle more tedious for your users.
+
+You can avoid this by specifying your dependencies. This will make sure that
+they are loaded in the kernel. It'll also make sure they are loaded *before*
+your own bundle, to make sure you can extend them.
+
+Additional use case for this is for distribution bundles, where one
+bundle is in fact bundling several others.
+
+Specifying Dependencies
+-----------------------
+
+Dependencies are specified using FQN_ (`Fully Qualified Name`) for the bundle class in
+same format as PHP uses for get_class_, and for class_ constant introduced in PHP 5.5.
+
+.. _FQN: https://en.wikipedia.org/wiki/Fully_qualified_name
+.. _get_class: http://php.net/manual/en/function.get-class.php
```

Comment:
please use ``phpfunction`` to link to the documentation of PHP builtin functions (similar to how ``class`` and ``method`` are used for Symfony classes and methods)